tn/export_md_to_pdf.py

```diff
@@ -187,9 +194,27 @@ def get_usfm_chunks(self):
 
             bible_dir = getattr(self, '{0}_dir'.format(resource))
             usfm = read_file(os.path.join(bible_dir, '{0}-{1}.usfm'.format(BOOK_NUMBERS[self.book_id],
-                                                                           self.book_id.upper())))
+                                                                           self.book_id.upper())), "utf-8") 
 
             chunks = re.compile(r'\\s5\s*\n*').split(usfm)
+
+            # TODO CROZ Make this an option
```

Comment:
Did you mean to leave this undone?

Reply:
Yes; it's work for the future.  It should be on the backlog instead of a comment in the source!  I'll fix that.

Reply:
Fixed.